user/quickshell/template.py

```diff
@@ -0,0 +1,67 @@
+pkgname = "quickshell"
+pkgver = "0.2.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [  # https://git.outfoxxed.me/quickshell/quickshell/src/branch/master/BUILD.md
+    '-DDISTRIBUTOR="Chimera Linux cports"',
+    "-DDISTRIBUTOR_DEBUGINFO_AVAILABLE=YES",
+    "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+    "-DCRASH_REPORTER=OFF",  # depends on google-breakpad
+    "-DUSE_JEMALLOC=ON",  # depends on jemalloc
+    "-DSOCKETS=ON",
+    "-DWAYLAND=ON",  # depends on qt6wayland, wayland, wayland-scanner, wayland-protocols
+    "-DWAYLAND_WLR_LAYERSHELL=ON",  # zwlr-layer-shell-v1 protocol
+    "-DWAYLAND_SESSION_LOCK=ON",  # ext-session-lock-v1 protocol
+    "-DWAYLAND_TOPLEVEL_MANAGEMENT=ON",  # zwlr-foreign-toplevel-management-v1 protocol
+    "-DSCREENCOPY=ON",  # depends on libdrm and libgdm
+    "-DSCREENCOPY_ICC=ON",  # ext-image-copy-capture-v1 protocol
+    "-DSCREENCOPY_WLR=ON",  # zwlr-screencopy-v1 protocol
+    "-DSCREENCOPY_HYPRLAND_TOPLEVEL=ON",  # hyprland-toplevel-export-v1 protocol
+    "-DX11=ON",  # depends on libxcb
+    "-DSERVICE_PIPEWIRE=ON",  # depends on libpipewire
+    "-DSERVICE_STATUS_NOTIFIER=ON",  # depends on qt6dbus
+    "-DSERVICE_MPRIS=ON",  # depends on qt6dbus
+    "-DSERVICE_PAM=ON",  # depends on pam
+    "-DHYPRLAND=ON",  # requires WAYLAND=ON
+    "-DHYPRLAND_GLOBAL_SHORTCUTS=ON",  # hyprland-global-shortcuts-v1 protocol
+    "-DHYPRLAND_FOCUS_GRAB=ON",  # hyprland-focus-grab-v1 protocol.
+    "-DI3=ON",
+    "-DI3_IPC=ON",
+]
```

Comment:
I left the configure settings to `ON` for Wayland and X11, as well as for Hyprland and I3; perhaps they should be subpackages instead so everything isn't compiled into one package? 

Something like the following:
```
quickshell
quickshell-hyprland
quickshell-x11
quickshell-i3
```